cmd/doctor.go

```diff
@@ -0,0 +1,78 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"runtime"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+	vfs "github.com/twpayne/go-vfs"
+)
+
+// doctorCmd represents the doctor command
+var doctorCommand = &cobra.Command{
+	Use:   "doctor",
+	Short: "Check your system for potential problems",
+	Long: `Check your system for potential problems. Such as:
+* Does the configuration file exist
+* Does the destination directory exist
+* Does the source directory exist
+* Is Hashicorp Vault installed?
+* Is Bitwarden installed?
+* Is LastPass installed?
+`,
+	RunE: makeRunE(config.runDoctorCommand),
+}
+
+func init() {
+	rootCommand.AddCommand(doctorCommand)
+
+}
+
+func (c *Config) runDoctorCommand(fs vfs.FS, args []string) error {
+	w := tabwriter.NewWriter(os.Stdout, 0, 0, 3, ' ', 0)
+
+	fmt.Fprintf(w, "Arch:\t%s\n", runtime.GOARCH)
+	fmt.Fprintf(w, "OS:\t%s\n", runtime.GOOS)
+
+	for _, dir := range []struct {
+		description string
+		path        string
+	}{
+		{description: "Configuration File", path: configFile},
+		{description: "Destination Directory", path: config.DestDir},
+		{description: "Source Directory", path: config.SourceDir},
+	} {
+		info, err := os.Stat(dir.path)
```

Comment:
Please use `fs.Stat`, not `os.Stat`. Also, I don't think `info` is used. You can write this as:
```go
    if _, err := fs.Stat(dir.path); err == nil {
        // print ok message
    } else {
        // print not ok message
    }
```

Reply:
info is used in all cases to print permissions if it exists